lib/service/index.ts

```diff
@@ -1,4 +1,5 @@
-import { buildSubgraphSchema } from '@apollo/federation';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { buildSubgraphSchema } from '@apollo/subgraph';
```

Comment:
remove this comment.

This file is used by the files exported by the lib, so it shouldn't use extraneous dependencies

Reply:
Thanks. I added it before moving the apollo libs to dependencies instead of devDependencies and forgot to remove after your comment.